content/02.main-text.md

```diff
@@ -24,36 +24,36 @@ When text is changed, Manubot automatically updates the manuscript's web page so
 Because manuscripts are created from GitHub repositories, Manubot supports a workflow where all edits are reviewed and discussed, ensuring that the collaborative text has a cohesive style and message and that authors receive precise credit for their work.
 These and other features provide support an open collaborative writing process that is not feasible with other writing platforms.
 
-## Manubot contribution workflow {#contribution-workflow}
+## Collaborative writing platforms
 
-There are many existing collaborative writing platforms.
-Some provide "what you see is what you get" (WYSIWYG) interfaces and support Microsoft Word or similar document formats.
-Others are LaTeX-based systems for technical writing [@doi:10.1038/514127a], such as [Overleaf](https://www.overleaf.com/) and [Authorea](https://www.authorea.com/).
-These tools ideally offer version control, multiple permission levels, or other functionality to support multi-author document editing.
-Existing platforms work well for editing text and are widely used for scholarly writing.
-However, they lack features that are important for open writing, such as managing a collaborative manuscript and attributing precise credit among many authors (Table @tbl:platforms).
-None of these platforms offer the ability to address thematically related changes together and enable multiple authors to iteratively refine proposed changes.
+There are many existing collaborative writing platforms (Table @tbl:platforms) [@doi:10.1038/514127a].
+In general, platforms with "what you see is what you get" (WYSIWYG) editors, such as Microsoft Word or Google Docs, require the least technical expertise to use.
+On the flip side, WYSIWIG platforms can be difficult to customize and incorporate into automated computational workflows.
+Traditionally, LaTeX has been used for these needs, since documents are written in plain text and the system is open source and customizable.
```

Comment:
` and customizable` feels redundant to me because `these needs` refers to customize and automate in the prior sentence.